src/GraphQL/Resolvers/EventStreamResolver.cs

```diff
@@ -15,6 +15,17 @@ public EventStreamResolver(Func<IResolveFieldContext, IObservable<TReturnType?>>
             _subscriber = context => new ValueTask<IObservable<object?>>((IObservable<object?>)subscriber(context));
         }
 
+        public EventStreamResolver(Func<IResolveFieldContext, ValueTask<IObservable<TReturnType?>>> subscriber)
+        {
+            if (subscriber == null)
```

Comment:
I'm waiting for C#7 and new syntax:
`  public EventStreamResolver(Func<IResolveFieldContext, ValueTask<IObservable<TReturnType?>>>!! subscriber)`

`!!` means that compiler generates null checks